po/fr.po

```diff
@@ -972,7 +972,7 @@ msgid ""
 "whitelist are able to change it."
 msgstr ""
 "Lockdown verrouille la configuration du pare-feu que seules les applications "
-"de la liste blanche peuvent modifiées."
+"de la liste blanche peuvent modifier."
```

Comment:
I have updated this string in zanata as per a native speaker.
The new string is:
"Lockdown verrouille la configuration du pare-feu afin que seules les applications de la liste blanche puissent la modifier."